scene/animate-images-with-image-overlay/README.metadata.json

```
@@ -0,0 +1,30 @@
+{
+    "category": "Scene",
+    "description": "Animate a series of images with an image overlay.",
+    "ignore": false,
+    "images": [
+        "AnimateImageWithImageOverlay.gif"
+    ],
+    "keywords": [
+        "3D",
+        "animation",
+        "drone",
+        "dynamic",
+        "image frame",
+        "image overlay",
+        "real time",
+        "rendering"
+    ],
+    "redirect_from": "/java/latest/sample-code/animate-images-with-image-overlay.htm",
+    "relevant_apis": [
+        "ImageFrame",
+        "ImageOverlay",
+        "SceneView"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/samples/animate_images_with_image_overlay/AnimateImagesWithImageOverlaySample.java",
+      "src/main/java/com/esri/samples/animate_images_with_image_overlay/AnimateImagesWithImageOverlayController.java",
+      "src/main/resources/animate_images_with_image)overlay.fxml"
+    ],
+    "title": "Animate Images With Image Overlay Sample"
```

I don't think we need to add the suffix 'sample' here
```suggestion
    "title": "Animate Images With Image Overlay"
```